src/app/components/RadioGruppe.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import React from 'react';
+import { Radio } from 'nav-frontend-skjema';
+
+export const satsRadioData = [
+    {
+        label: '80%',
+        value: 80
+    },
+    {
+        label: '100%',
+        value: 100
+    }
+];
+
+export const omsorgRadioData = [
+    {
+        label: 'Felles omsorg',
+        value: 'Felles omsorg'
+    },
+    {
+        label: 'Aleneomsorg',
+        value: 'Aleneomsorg'
+    }
+];
+
+type Props = {
+    listOfRadioData: Object,
+    name: string
+}
+
+const RadioGruppe = (props: Props) => (
```

Comment:
Ingenting domenespesifikt for denne, så tenker det ville være bedre å kalle denne for RadioGroup, slik at vi holder oss konsekvente på norsk / engelsk kildekode.